### ISSUE TITLE
Build error with PyPI install

### ISSUE THREAD

**Issue by levitsky** [mobiusklein/pyteomics.cythonize#5]
I know it was only yesterday that I said that PyPI install without Cython worked for me, but it doesn't anymore.

On a fresh Arch Linux system with gcc 11.1.0 I get an error when building the wheel from `pyteomics.cythonize-0.2.2.tar.gz`.

The command: `pip install pyteomics.cythonize --no-cache-dir` in a fresh `Python 3.9.5` virtualenv with `pip==21.1.2, setuptools==57.0.0, wheel==0.36.2`.

The error:
```
building 'pyteomics.cmass' extension                                                                                                                                                                           
  gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -march=x86-64 -mtune=generic -O3 -pipe -fno-plt -fno-semantic-interposition -march=x86-64 -mtune=generic -O3 -pipe -fno-plt -march=x86-64 -mtune=generic -O3 -pipe -fno-plt -fPIC -I/home/lev/.virtualenvs/nocython/include -I/usr/include/python3.9 -c pyteomics/cmass.c -o build/temp.linux-x86_64-3.9/pyteomics/cmass.o                         
  pyteomics/cmass.c: In function ‘__Pyx_modinit_type_init_code’:                                                                                                                                                 
  pyteomics/cmass.c:15608:44: error: ‘PyTypeObject’ {aka ‘struct _typeobject’} has no member named ‘tp_print’                                                                                                    
  15608 |   __pyx_type_9pyteomics_5cmass_CComposition.tp_print = 0;                                                                                                                                              
        |                                            ^             
```
When I cythonize `cmass.pyx` with latest stable Cython, I see the assignment wrapped with a version check which I think is equivalent to `Python < 3.8.0b1`:
```
  #if PY_VERSION_HEX < 0x030800B1
  __pyx_type_9pyteomics_5cmass_CComposition.tp_print = 0;
```
There is no such check in the packaged `cmass.c`, which may be the cause of the problem. I suppose that updating the C files in the source package with latest Cython should fix this?

**Comment by mobiusklein** [comment]
You're right. The last release was a two years ago, and Cython has adapted to changes to the CPython C API to account for breaking changes. I'll throw together the CD machinery and tag a new release.

**Comment by mobiusklein** [comment]
Just published a new set of released (v0.2.3) with manylinux wheels  and a source distribution. Please let me know if this doesn't resolve the issue.

**Comment by levitsky** [comment]
It certainly resolves the issue in the sense that I don't have to build the wheel locally, and the provided wheel installs fine.
However, building from the packaged source tarball still fails:

```
building 'pyteomics.cparser' extension
  creating build/temp.linux-x86_64-3.9
  creating build/temp.linux-x86_64-3.9/pyteomics
  gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -march=x86-64 -mtune=generic -O3 -pipe -fno-plt -fno-semantic-interposition -march=x86-64 -mtune=generic -O3 -pipe -fno-plt -march=x86-64 -mtune=generic -O3 -pipe -fno-plt -fPIC -I/home/lev/.virtualenvs/nocython/include -I/usr/include/python3.9 -c pyteomics/cparser.c -o build/temp.linux-x86_64-3.9/pyteomics/cparser.o
  pyteomics/cparser.c:631:10: fatal error: compat.h: No such file or directory
    631 | #include "compat.h"
        |          ^~~~~~~~~~
  compilation terminated.
  error: command '/usr/bin/gcc' failed with exit code 1
  ----------------------------------------
  ERROR: Failed building wheel for pyteomics.cythonize
```

**Comment by mobiusklein** [comment]
Just released v0.2.4 with an updated manifest that should fix the problem.

**Comment by levitsky** [comment]
Yep, all good now. Thanks!
